### DISCUSSION TITLE
Contributing to RocketCHat

### DISCUSSION THREAD

**Discussion by preyanshu** [RocketChat/Rocket.Chat#31133] [Q&A]
Hi, I'm Preyanshu, a second-year undergrad at NITH, specializing in full-stack MERN. I'm very much interested in contributing to Rocket.Chat. Can Someone please guide me from where should i start based on my current knowledge of fullstack mern development? Apologies if this is a very basic question, I'm new to open source contributions.

**Comment by GlenLamb** [comment]
Reproducing a reported bug
You can contribute to an open source project by validating an issue or adding additional context to an existing issue.

Testing a pull request
You can contribute to an open source project by merging a pull request into your local copy of the project and testing the changes. Add the outcome of your testing in a comment on the pull request.

Updating issues
You can contribute to an open source project by adding additional information to existing issues.
